cmd/gb-vendor/vendor/manifest.go

```diff
@@ -137,3 +141,14 @@ func readManifest(r io.Reader) (*Manifest, error) {
 	err := d.Decode(&m)
 	return &m, err
 }
+
+// Implement sort for dependencies
```

Comment:
I'm a little lazy for unexported types. Feel free to fix it or I could go back and do so if you feel strongly.